modules/styling/stories/Basics.stories.mdx

```diff
@@ -312,3 +312,55 @@ The HTML will look something like this:
 Notice the stencil adds all the class names that match the base, modifiers, and compound modifiers.
 
 <ExampleCodeBlock code={CreateStencil} />
+
+### Keyframes
+
+The `keyframes` function re-exports the [Emotion CSS keyframes](https://emotion.sh/docs/keyframes)
+function, but is compatible with a custom Emotion instance and is understood by the Static style
+transformer.
+
```

Comment:
Should you also call out the injectGlobal function

Reply:
I don't have support in the style transform yet, but I think it makes sense to add it.

Reply:
Added it.